pkgs/development/compilers/djgpp/default.nix

```
@@ -0,0 +1,95 @@
+{ bison
+, buildPackages
+, curl
+, fetchFromGitHub
+, fetchurl
+, file
+, flex
+, i686Build ? false
```

A suggestion: it is better to use a string here instead of a Boolean:


```suggestion
, targetArchitecture ? "i586"
```